model_specific/SM/standard_model.hpp

```diff
@@ -657,6 +654,7 @@ class Standard_model : public Beta_function {
    Eigen::Array<double,3,1> MFu{Eigen::Array<double,3,1>::Zero()};
    Eigen::Array<double,3,1> MFe{Eigen::Array<double,3,1>::Zero()};
    double MVWp{};
+   double MVWp_pole{};
```

Comment:
Please add a comment: `// W boson pole mass (determined from fit formula)` or similar, to avoid confusion.

Reply:
Done in 42e877323e3f57726a8e46137890990cdddba3f5.

Reply:
Sorry, would you mind changing the comment to
~~~.cpp
double MVWp_pole{}; ///< W boson pole mass (determined from fit formula)
~~~
in order for Doxygen to generate a corresponding entry? Sorry for being pedantic. :)

Reply:
(Also for `MVWp` above.)